pkgs/development/libraries/criterion/default.nix

```diff
@@ -30,7 +35,12 @@ stdenv.mkDerivation rec {
   preCheck = ''
     export LD_LIBRARY_PATH=`pwd`''${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH
   '';
-  checkTarget = "criterion_tests test";
+  checkTarget = "test";
+
+  patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Might use `preConfigure` like the boxfort package did for consistency